1.0/tools/tests.md

```diff
@@ -0,0 +1,421 @@
+Check out Sinon's documentation for more in-depth examples. 
+
+{% highlight html %}
+<!-- create test fixture template -->
+<test-fixture id="simple-get">
+  <template>
+    <iron-ajax url="/responds_to_get_with_json"></iron-ajax>
+  </template>
+</test-fixture>
+<script>
+  suite('<iron-ajax>', function() {
+    var ajax;
+    var request;
+    var server;
+    var responseHeaders = {
+      json: { 'Content-Type': 'application/json' }
+    };
+    setup(function() {
+      server = sinon.fakeServer.create();
+      server.respondWith(
+        'GET',
+        /\/responds_to_get_with_json.*/, [
+          200,
+          responseHeaders.json,
+          '{"success":true}'
+        ]
+      );
+    });
+    teardown(function() {
+      server.restore();
+    });
+    suite('when making simple GET requests for JSON', function() {
+      setup(function() {
+        // get fresh instance of iron-ajax before every test
+        ajax = fixture('simple-get');
+      });
+      test('has sane defaults that love you', function() {
+        request = ajax.generateRequest();
+        server.respond();
+        expect(request.response).to.be.ok;
+        expect(request.response).to.be.an('object');
+        expect(request.response.success).to.be.equal(true);
+      });
+      test('has the correct xhr method', function() {
+        request = ajax.generateRequest();
+        expect(request.xhr.method).to.be.equal('GET');
+      });
+    });
+  });
+</script>
+{% endhighlight %}
+
+**Note:** The example above uses Chai's [`expect`](http://chaijs.com/api/bdd/) 
+assertion style. 
+{: .alert .alert-info }
+
+## Run a set of tests {#test-sets}
+
+To run a set of tests, create an HTML file and call `loadSuites()`. When
+running `wct`, specify the path to the HTML file as the first argument 
+(for example, `wct test/my-test-set.html`.
+
+{% highlight html %}
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <script src=”../bower_components/webcomponentsjs/web-components-lite.js”></script>
+    <script src=”../bower_components/web-component-tester/browser.js”></script>
+  </head>
+  <body>
+    <script>
+      WCT.loadSuites([
+        'basic.html',
+        'async.html'
+      ]);
+    </script>
+  </body>
+</html>
+{% endhighlight %}
+
+The argument to `loadSuites()` should be an array of strings. Each string
+should be a relative URL to a test suite. You can configure your tests 
+using query strings in the URLs. See [Test shadow DOM](#shadow-dom)
+for an example.
+
+## Test local DOM
+
+Use Polymer's [DOM API](/1.0/docs/devguide/local-dom.html#dom-api) to access 
+and modify local DOM children.
+
+    test('click sets isWaiting to true', function() {
+      myEl.$$('button').click();
+      assert(myEl.isWaiting, true);
+    });
+   
+**Note:** `myEl.$$('button')` returns the first `button` element encountered
+in the local DOM of `myEl`.
+{: .alert .alert-info }
+ 
+### DOM mutations
```

Comment:
Sorry, I wasn't very clear on my previous comment. I just meant to insert the word "DOM":
"Test asynchronous DOM mutations". You could leave out the "asynchronous" if you want, but for consistency, should keep the "Test." Sorry for the confusion.